starter-code/json-hw.js

```diff
@@ -19,14 +18,21 @@ with the date the photo was taken. The first console.log should return the follo
 
 */
 
+flickerObj.element.forEach(function(elem){
+    console.log(title + " . " + date_taken)
```

Comment:
`flickerObj.items` it's an array of object so you should use `elem` to access any key in it.
```suggestion
    console.log(elem.title + " . " + elem.date_taken)
```